cherrypicks_test.go

```diff
@@ -217,6 +221,7 @@ Hello :smile_cat: This PR contains changelog entries. Please, verify the need of
 			conf.integrationDirectory = tmpdir
 
 			log := logrus.NewEntry(logrus.StandardLogger())
+			log.Infof(" TEST: %s", name)
```

Comment:
@lluiscampos ****, this line slipped through my eye holes. Make a new PR to revert this line? Woopz

Reply:
I saw it but didn't react neither. It doesn't bother me so feel free to leave it. As you wish.